settings/clientSettings.ts

```diff
@@ -34,6 +34,9 @@ export const BAKED_GRAPHER_EXPORTS_BASE_URL: string =
 export const ADMIN_BASE_URL: string =
     process.env.ADMIN_BASE_URL ??
     `http://${ADMIN_SERVER_HOST}:${ADMIN_SERVER_PORT}`
+// e.g. "https://catalog.ourworldindata.org/baked-variables/live_grapher/", fallback to
+// BAKED_GRAPHER_URL if not given
+export const DATA_BASE_URL: string = process.env.DATA_BASE_URL ?? ""
```

Comment:
You can do the fallbacking in here already:
```suggestion
export const DATA_BASE_URL: string = process.env.DATA_BASE_URL ?? `${BAKED_GRAPHER_URL ?? ""}/data/variables/`
```